### PR TITLE
[DOCS] Docs: Minor improvements

### PR DESCRIPTION
**Pull request by M-arcus**

### CODE REVIEW

**Comment by M-arcus** [comment]
[WIP] Please do not merge yet, there are more commits coming

File: doc/Addons.md [hunk 1]
```diff
@@ -321,184 +321,299 @@ Complete list of hook callbacks
...
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_post', $_POST);
+    Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
+    Addon::callHooks($a->module.'_mod_content', $arr);
+    Addon::callHooks($a->module.'_mod_aftercontent', $arr);
+    Addon::callHooks('page_end', $a->page['content']);
+    
+### include/api.php
+	Addon::callHooks('logged_in', $a->user);
```
Comment (tobiasd) [inline doc/Addons.md]: Should these be indented as well and prefaced by an empty line.

Comment (M-arcus) [reply doc/Addons.md]: Fixed

File: doc/de/Addons.md [hunk 2]
```diff
@@ -191,184 +191,300 @@ Komplette Liste der Hook-Callbacks
...
+    Addon::callHooks($a->module.'_mod_init', $placeholder);
+    Addon::callHooks($a->module.'_mod_post', $_POST);
+    Addon::callHooks($a->module.'_mod_afterpost', $placeholder);
+    Addon::callHooks($a->module.'_mod_content', $arr);
+    Addon::callHooks($a->module.'_mod_aftercontent', $arr);
+    Addon::callHooks('page_end', $a->page['content']);
+    
+### include/api.php
+
+	Addon::callHooks('logged_in', $a->user);
```
Comment (MrPetovan) [inline doc/de/Addons.md]: The indentation style still are mixed, in one case it is a tab, in the other 4 spaces. Please choose one of the other for the whole file.

Comment (tobiasd) [reply doc/de/Addons.md]: I think it should be consistent even broader, for the DE and EN versions of the file alike.

Comment (M-arcus) [reply doc/de/Addons.md]: Fixed

**Pull request merged**